cg/models/cg_config.py

```diff
@@ -257,6 +257,7 @@ class CGConfig(BaseModel):
     environment: Literal["production", "stage"] = "stage"
     flow_cells_dir: str
```

Comment:
```suggestion
    illumina_flow_cells_dir: str
```
perhaps also?

Reply:
Do you think this should be `illumina_data_directory`?

Reply:
I guess it makes sense with the proposed changes in the issue you set up. having both directories under the "sequencing_data" umbrella. Then they could also simply be "illumina" and "nanopore". I don't mind one way or another, just a unified way representing raw sequencing output data would be nice I think

Reply:
Agreed, will change it when we've decided on the folder structure 👍